src/refmvs.rs

```diff
@@ -85,15 +85,28 @@ pub struct refmvs_mvpair {
     pub mv: [mv; 2],
 }
 
+/// For why this unaligned, see the aligned [`refmvs_block`] below.
 #[derive(Copy, Clone)]
 #[repr(C, packed)]
-pub struct refmvs_block {
+pub struct refmvs_block_unaligned {
```

Comment:
Does the definition need to be `pub`? It could be `pub(crate)`.

Reply:
It could be `pub(crate)`, but basically everything is still `pub` at this point whether it needs to be or not, so I haven't been fixing those at all.